instagram-app/src/components/CommentSection/CommentSection.js

```diff
@@ -1,20 +1,69 @@
 import React from 'react';
 import Comment from './Comment.js';
+import { FaEllipsisH } from 'react-icons/fa';
 
-function CommentSection(props) {
-    return (
-        <div>
-            {props.comments.map(comment => (
-                <Comment
-                key={comment.id}    
-                username={comment.username}
-                text={comment.text} 
-                />
-            ))}
-            <input type="text" placeholder="Add a comment..." />
-            <p>time here</p>
-        </div>
-    )
+
+class CommentSection extends React.Component {
+    
+    state = {
+        comments: [],
+        comment: "",
+        username: ""
+    }
+
+    componentDidMount() {
+        // fake API call here
+            // 1. call API
+            // 2. API returns data
+            // 3. set data to state
+        this.setState({
+            comments: this.props.comments,
+            comment: "",
+            username: "bill"
+        })
+    }
```

Comment:
Don't forget to clean up after developing, You wouldn't want this to make it into production.